lib/cisco_node_utils/cmd_ref/DEPRECATED.yaml

```diff
@@ -15,6 +15,17 @@ private_vlan_any:
   multiple:
```

Comment:
No, I decided to just put all of the deprecated properties in the same yaml file since they're unique. I don't see an advantage to splitting them into separate files.